### PR TITLE
The Event Logic of the Math Magician is Completed

### PR DESCRIPTION
**Pull request by ometman**
The event logic for the project has been completed.
 - A new directory, logic' was created and used to hold calculate.js and operate.js.
 - Big.js was installed
 - Calculate.js was import into the Calculator.js component
 - Implement click event handlers using the math logic from calculate.js and operate.js in the React component
 - Calculator functions correctly in the browser in performing math operations.
 - Linters checked and passed.

### CODE REVIEW

File: src/logic/Calculate.js
Comment (Efiamotu-1) [inline src/logic/Calculate.js:1]: - [ ] When i try to use the divide operator i am faced with the error shown below, please kindly fix this error and resubmit your project for approval :+1: 
iu

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ometman  

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/Calculate.js
Comment (ometman) [reply src/logic/Calculate.js:1]: Whoosh, I missed the innerText value. 😄 
It is fixed. Thanks 🙏

Comment (Alusp) [inline src/logic/Calculate.js:1]: - [ ] Very good job in resolving the division symbol, :clap: , However, as mentioned by the previous reviewer, When I try to use the Multiplication and the Subtraction operators i am faced with the error shown below, please kindly fix this error and resubmit your project for approval 👍

<table>
    <tr>
      <td>Multiplication</td>
      <td>Subtraction</td>
    </tr>
    <tr>
      <td>

StudentMultiply

</td>
      <td>

MinusStudent

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @ometman, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✔️ No linter Errors
✔️ Descriptive README
✔️ Descriptive PR title and description
✔️ Completed most of the Tasks


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/Calculate.js
Comment (ometman) [reply src/logic/Calculate.js:1]: I can't tell when you reviewed, it was fixed about an hour ago.

Comment (prolajumokeoni) [reply src/logic/Calculate.js:1]: Hi @ometman, this issue is fixed 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @ometman 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - Calculator app works properly ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ometman** [comment]
Thanks @prolajumokeoni

**Pull request merged**